languages/python/exercises/concept/pretty-leaflet/.docs/introduction.md

```diff
@@ -0,0 +1,79 @@
+## string-formatting
+
+The Python `str` build-in type can be initialized using two robust string formatting methods. The string interpolation with `f'{variable}'` should be preferred as it is very readable, complete, and very fast. When a more versatile approach is needed, `''.format()` allows creating almost all the `str` you need.
```

Comment:
```suggestion
The Python `str` built-in can be initialized using two robust string formatting methods :  `f-strings` and `str.format()`. String interpolation with `f'{variable}'` is preferred because it is easily read, complete in place, and very fast. When an internationalization-friendly or more flexible approach is needed, `str.format()` allows creating almost all the other `str` variations you might  need.
```